src/app/portlets/dot-edit-page/layout/dot-edit-layout-grid/dot-edit-layout-grid.component.spec.ts

```diff
@@ -17,57 +23,249 @@ class TestActionButtonComponent {
+                    }
+                },
+                '6a12bbda-0ae2-4121-a98b-ad8069eaff3a': {
+                    container: {
+                        type: 'containers',
+                        identifier: '6a12bbda-0ae2-4121-a98b-ad8069eaff3a',
+                        name: 'Banner Carousel ',
+                        categoryId: '427c47a4-c380-439f-a6d0-97d81deed57e'
+                    }
+                },
+                'a6e9652b-8183-4c09-b775-26196b09a300': {
+                    container: {
+                        type: 'containers',
+                        identifier: 'a6e9652b-8183-4c09-b775-26196b09a300',
+                        name: 'Default 4 (Page Content)',
+                        categoryId: '8cbcb97e-8e04-4691-8555-da82c3dc4a91'
+                    }
+                },
+                'd71d56b4-0a8b-4bb2-be15-ffa5a23366ea': {
+                    container: {
+                        type: 'containers',
+                        identifier: 'd71d56b4-0a8b-4bb2-be15-ffa5a23366ea',
+                        name: 'Blank Container',
+                        categoryId: '3ba890c5-670c-467d-890d-bd8e9b9bb5ef'
+                    }
+                }
+            },
+            page: null,
+            layout: {
+                title: 'anonymouslayout1511296039453',
+                header: false,
+                footer: false,
+                body: {
+                    rows: [
+                        {
+                            columns: [
+                                {
+                                    containers: ['56bd55ea-b04b-480d-9e37-5d6f9217dcc3'],
+                                    leftOffset: 1,
+                                    width: 8
+                                },
+                                {
+                                    containers: ['5363c6c6-5ba0-4946-b7af-cf875188ac2e'],
+                                    leftOffset: 9,
+                                    width: 4
+                                }
+                            ]
+                        },
+                        {
+                            columns: [
+                                {
+                                    containers: [
+                                        'd71d56b4-0a8b-4bb2-be15-ffa5a23366ea',
+                                        'a6e9652b-8183-4c09-b775-26196b09a300'
+                                    ],
+                                    leftOffset: 1,
+                                    width: 3
+                                },
+                                {
+                                    containers: ['6a12bbda-0ae2-4121-a98b-ad8069eaff3a'],
+                                    leftOffset: 4,
+                                    width: 3
+                                }
+                            ]
+                        }
+                    ]
+                },
+                sidebar: null
+            }
+        };
```

Comment:
Really do we need a too big object for this test?

Reply:
yes, I tried to reduce as much as possible.